include/dlaf/eigensolver/tridiag_solver/merge.h

```diff
@@ -25,6 +25,7 @@
 #include <dlaf/communication/communicator_pipeline.h>
 #include <dlaf/communication/index.h>
 #include <dlaf/communication/kernels.h>
+#include <dlaf/communication/kernels/internal/broadcast.h>
```

Comment:
Note: needed for https://github.com/eth-cscs/DLA-Future/blob/691794b11e472ee337ccb9e19122a42ae35406f6/include/dlaf/eigensolver/tridiag_solver/merge.h#L1269-L1270 as the function objects has been moved in the internal file.